internal/service/alarms/api/openapi.yaml

```diff
@@ -221,6 +222,106 @@ paths:
               schema:
                 $ref: '../../common/api/openapi.yaml#/components/schemas/ProblemDetails'
 
+  /o2ims-infrastructureMonitoring/v1/alarmServiceConfiguration:
```

Comment:
sorry think i missed the discussion...but did we decide this already? basically for phase 1 we were going to hardcode the retention period to 24h. And make it configureable later on. Alternatively instead of exposing this in api we could pass this info in the innovatory CR or configMap that we can we reload right?

if already decided lgtm but im totally out of the loop! fyi @browsell

Reply:
This is coming from the specification.

3.3.4.6 REST resource: Alarm Service Configuration (O2IMS-INTERFACE-R004-v07.00)
Table 4.2.1.4.13.2-1 Attributes Properties for AlarmList (O-CLOUD-IM.0-R004-v02.00)
3.7.13 Alarm Purge Use Case (ORCH-USE-CASES-R003-v11.00)